README.md

```diff
@@ -2,18 +2,15 @@
 
 Proposal to define an XML-based logging format for outputs from EDA tools and logging libraries.
 
-**Report Kinds**  
-* [Test Report Summary](#test-report-summary)
```

Comment:
Don't we need a ToC or does GH Markdown support a ToC instruction like GitLab?

Reply:
As commented in a meeting, GitHub provides a ToC for any markdown source.: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/.